src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
> Doesn't "doubly-linked" sort of imply that it is circular

No, and I'm on my phone right now but if memory serves this implementation isn't circular. Circular means that you can't walk off the end, that you can follow the next (or prev) pointers forever rather than eventually hitting null.